core/modules/replica/HttpIngestChunksModule.h

```diff
@@ -110,7 +110,7 @@ class HttpIngestChunksModule: public HttpModule {
                            std::string const& database,
                            unsigned int chunk) const;
 
-    util::Mutex _ingestManagementMtx;   /// Synchronized access to the Ingest management operations
+    static util::Mutex _ingestManagementMtx;    /// Synchronized access to the chunk management operations
```

Comment:
Maybe a few less spaces between the line and the comment.

Reply:
👍